cdc/api/v1/api_test.go

```diff
@@ -435,6 +436,112 @@ func TestRebalanceTables(t *testing.T) {
 	require.Contains(t, respErr.Error, "changefeed not exists")
 }
 
+func TestDrainCapture(t *testing.T) {
+	t.Parallel()
+
+	statusProvider := newStatusProvider()
+
+	ctrl := gomock.NewController(t)
+	owner := mock_owner.NewMockOwner(ctrl)
+	capture := capture.NewCapture4Test(owner)
+	router := newRouter(capture, statusProvider)
+
+	captureInfo, err := capture.Info()
+	require.NoError(t, err)
+	data := model.DrainCaptureRequest{
+		CaptureID: captureInfo.ID,
+	}
+	b, err := json.Marshal(&data)
+	require.NoError(t, err)
+
+	body := bytes.NewReader(b)
+	api := testCase{
+		url:    "/api/v1/captures/drain",
+		method: "POST",
```

Comment:
It occurs to me, we should use `PUT` as it idempotent [1]

\[1\]: https://restfulapi.net/rest-put-vs-post/